### PR TITLE
Lint and docs

### PR DESCRIPTION
**Pull request by SimonSchick**
linting and docs for sanity

Also it seems airbnb introduced some funky rules (which totally break javascript).

In the future pls provide basic doctags so we don't have to reverse engineer code 😄

### CODE REVIEW

**Comment by connor4312** [comment]
AirBnB's linting rules have been a pain to implement and keep up with in the past, with big changes in every major version, sometimes breaking changes in minor versions, etc. Thoughts on using [standard](https://github.com/feross/standard) or semistandard instead?

**Comment by SimonSchick** [comment]
No semicolons is kinda weird imo, tbh we I'd rather have our own styleguide at this point to be honest.

**Comment by SimonSchick** [comment]
I'm actually more inclined to rewrite this in TS with build target es6 instead, that good?

**Comment by SimonSchick** [comment]
I removed eslint from now, I'd still like to get this in so we have documentation.

File: index.js [hunk 1]
```diff
@@ -132,7 +173,7 @@ function strip(ast, options) {
         || (data.name !== options.value && data.negated);
 
     switch (data.directive) {
-    case 'start':
+    case 'start': {
```
Comment (connor4312) [inline index.js]: What, this is bizarre. Can we not? 😛

Comment (SimonSchick) [inline index.js]: Right, that was one of `eslint`s complaints because of scoping and such, I'll remove it if it bothers you so much :P

**Comment by SimonSchick** [comment]
Happy @connor4312 ?

**Comment by connor4312** [comment]
Same thing on the `start` block.

Re TS: feel free to rewrite it but initially I didn't think it necessary for a ~200 line script.

**Comment by SimonSchick** [comment]
It is not, but in the future I feel like it's better to simply go with TS as it is self documenting, adds type safety by default, JS has neither and the overhead of using typescript is not very high.

**Pull request merged**